### ISSUE TITLE
Load Error with 4.6+ (dp parameter is null)

### ISSUE THREAD

**Issue by nestyle** [eirannejad/pyRevit#419]
To get pyrevit to load without an error I have to disable modplus plugin.
In the log below there is reference to a different user name.
"C:\Users\LeoW10\AppData\Roaming\pyRevit\pyRevit\dev\pyRevitLoader\Source\ScriptExecutor.cs"

``` vba
' 0:< TaskDialog "Traceback (most recent call last):
'  File "C:\Users\---\AppData\Roaming\pyRevit-Master\bin\engines\pyRevitLoader.py", line 32, in <module>
'  File "C:\Users\---\AppData\Roaming\pyRevit-Master\pyrevitlib\pyrevit\loader\sessionmgr.py", line 215, in load_session
'  File "C:\Users\---\AppData\Roaming\pyRevit-Master\pyrevitlib\pyrevit\loader\sessionmgr.py", line 64, in _setup_output
'TypeError: Value cannot be null.
'Parameter name: dp
'
'
'System.ArgumentNullException: Value cannot be null.
'Parameter name: dp
'   at Microsoft.Scripting.Interpreter.NewInstruction.Run(InterpretedFrame frame)
'   at Microsoft.Scripting.Interpreter.Interpreter.Run(InterpretedFrame frame)
'   at Microsoft.Scripting.Interpreter.LightLambda.Run3[T0,T1,T2,TRet](T0 arg0, T1 arg1, T2 arg2)
'   at System.Dynamic.UpdateDelegates.UpdateAndExecute2[T0,T1,TRet](CallSite site, T0 arg0, T1 arg1)
'   at IronPython.Compiler.Ast.CallExpression.Invoke0Instruction.Run(InterpretedFrame frame)
'   at Microsoft.Scripting.Interpreter.Interpreter.Run(InterpretedFrame frame)
'   at Microsoft.Scripting.Interpreter.LightLambda.Run1[T0,TRet](T0 arg0)
'   at System.Dynamic.UpdateDelegates.UpdateAndExecute2[T0,T1,TRet](CallSite site, T0 arg0, T1 arg1)
'   at IronPython.Compiler.Ast.CallExpression.Invoke0Instruction.Run(InterpretedFrame frame)
'   at Microsoft.Scripting.Interpreter.Interpreter.Run(InterpretedFrame frame)
'   at Microsoft.Scripting.Interpreter.LightLambda.Run1[T0,TRet](T0 arg0)
'   at System.Dynamic.UpdateDelegates.UpdateAndExecute2[T0,T1,TRet](CallSite site, T0 arg0, T1 arg1)
'   at Microsoft.Scripting.Interpreter.DynamicInstruction`3.Run(InterpretedFrame frame)
'   at Microsoft.Scripting.Interpreter.Interpreter.Run(InterpretedFrame frame)
'   at Microsoft.Scripting.Interpreter.LightLambda.Run2[T0,T1,TRet](T0 arg0, T1 arg1)
'   at IronPython.Compiler.PythonScriptCode.RunWorker(CodeContext ctx)
'   at Microsoft.Scripting.Hosting.ScriptSource.Execute(ScriptScope scope)
'   at PyRevitLoader.ScriptExecutor.ExecuteScript(String sourcePath) in C:\Users\LeoW10\AppData\Roaming\pyRevit\pyRevit\dev\pyRevitLoader\Source\ScriptExecutor.cs:line 70"
'CommonButtons : Close
'DefaultButton : Close 
' 0:< ::36:: Delta VM: Avail +2 -> 134215735 MB, Used 396 MB; RAM: Avail -15 -> 28414 MB, Used +3 -> 590 MB, Peak +3 -> 591 MB 
' 0:< GUI Resource Usage GDI: Avail 9772, Used 228, User: Used 210 
'H 08-Nov-2018 10:35:18.322;   0:< 
Jrn.Data "TaskDialogResult"  _
        , "Traceback (most recent call last):" & vbLf & "  File ""C:\Users\---\AppData\Roaming\pyRevit-Master\bin\engines\pyRevitLoader.py"", line 32, in <module>" & vbLf & "  File ""C:\Users\---\AppData\Roaming\pyRevit-Master\pyrevitlib\pyrevit\loader\sessionmgr.py"", line 215, in load_session" & vbLf & "  File ""C:\Users\---\AppData\Roaming\pyRevit-Master\pyrevitlib\pyrevit\loader\sessionmgr.py"", line 64, in _setup_output" & vbLf & "TypeError: Value cannot be null." & vbLf & "Parameter name: dp" & vbLf & "" & vbLf & "" & vbLf & "System.ArgumentNullException: Value cannot be null." & vbLf & "Parameter name: dp" & vbLf & "   at Microsoft.Scripting.Interpreter.NewInstruction.Run(InterpretedFrame frame)" & vbLf & "   at Microsoft.Scripting.Interpreter.Interpreter.Run(InterpretedFrame frame)" & vbLf & "   at Microsoft.Scripting.Interpreter.LightLambda.Run3[T0,T1,T2,TRet](T0 arg0, T1 arg1, T2 arg2)" & vbLf & "   at System.Dynamic.UpdateDelegates.UpdateAndExecute2[T0,T1,TRet](CallSite site, T0 arg0, T1 arg1)" & vbLf & "   at IronPython.Compiler.Ast.CallExpression.Invoke0Instruction.Run(InterpretedFrame frame)" & vbLf & "   at Microsoft.Scripting.Interpreter.Interpreter.Run(InterpretedFrame frame)" & vbLf & "   at Microsoft.Scripting.Interpreter.LightLambda.Run1[T0,TRet](T0 arg0)" & vbLf & "   at System.Dynamic.UpdateDelegates.UpdateAndExecute2[T0,T1,TRet](CallSite site, T0 arg0, T1 arg1)" & vbLf & "   at IronPython.Compiler.Ast.CallExpression.Invoke0Instruction.Run(InterpretedFrame frame)" & vbLf & "   at Microsoft.Scripting.Interpreter.Interpreter.Run(InterpretedFrame frame)" & vbLf & "   at Microsoft.Scripting.Interpreter.LightLambda.Run1[T0,TRet](T0 arg0)" & vbLf & "   at System.Dynamic.UpdateDelegates.UpdateAndExecute2[T0,T1,TRet](CallSite site, T0 arg0, T1 arg1)" & vbLf & "   at Microsoft.Scripting.Interpreter.DynamicInstruction`3.Run(InterpretedFrame frame)" & vbLf & "   at Microsoft.Scripting.Interpreter.Interpreter.Run(InterpretedFrame frame)" & vbLf & "   at Microsoft.Scripting.Interpreter.LightLambda.Run2[T0,T1,TRet](T0 arg0, T1 arg1)" & vbLf & "   at IronPython.Compiler.PythonScriptCode.RunWorker(CodeContext ctx)" & vbLf & "   at Microsoft.Scripting.Hosting.ScriptSource.Execute(ScriptScope scope)" & vbLf & "   at PyRevitLoader.ScriptExecutor.ExecuteScript(String sourcePath) in C:\Users\LeoW10\AppData\Roaming\pyRevit\pyRevit\dev\pyRevitLoader\Source\ScriptExecutor.cs:line 70",  _
         "Close", "IDCLOSE"
' 0:< API_SUCCESS { Starting External Application: PyRevitLoader, Class: PyRevitLoader.PyRevitLoaderApplication, Vendor : eirannejad(), Assembly: C:\Users\---\AppData\Roaming\pyRevit-Master\bin\engines\277\pyRevitLoader.dll }
```

_Originally posted by @nestyle in https://github.com/eirannejad/pyRevit/issues/413#issuecomment-437244107_

**Comment by eirannejad** [comment]
@nestyle Thanks for creating this report.

**Comment by DmytroSE** [comment]
Hi! Is there any solution for this issue? ModPlus plugin is very helpfull, but pyrevit doesn't work with it

**Comment by nestyle** [comment]
There was an update to modplus recently, it seems to have maybe fixed the conflict.
I tested it with pyrevit 4.6.8.

**Comment by eirannejad** [comment]
Awesome! Thanks for the update @nestyle

**Comment by eirannejad** [comment]
A new dll finder has been added to pyRevit that would direct Revit to load missing dependencies from pyRevit bin path. It will be published with 4.6.9 this weekend. Please test and reopen if the issue persists
